vue-components/src/components/DateInput.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+	<div
+		:class="[ 'wikit', 'wikit-DateInput' ]"
+	>
+		<InputWithExtender
+			:label="label"
+			:value="value"
+			:feedback-type="feedbackType"
+			:placeholder="placeholder"
+			:disabled="disabled"
+			@input="$emit( 'input', $event )"
+		>
+			<template v-slot:default>
+				<slot v-if="!value" name="prompt" />
+				<p class="wikit-DateInput--IntroText" v-else>
+					{{ resultsIntroText }}
+				</p>
+				<p class="wikit-DateInput--ParsedValue" v-if="parsedValue && value">
+					<b>{{ parsedValue }}</b>
+				</p>
+				<BouncingDots v-if="!parsedValue && value" type="small" />
+				<div class="wikit-DateInput--CalendarNotice" v-if="calendarNotice && parsedValue">
```

Comment:
I think this class should be `wikit-DateInput__CalendarNotice` - it is an element, not a modifier, right?